docs/development/compatibility-policy.rst

```diff
@@ -573,3 +608,41 @@ Tests which need to use deprecated classes should use the :py:meth:`getDeprecate
             creds = UsernameHashedPassword(b"foo", b"bar")
             self.assertEqual(creds.username, b"foo")
             self.assertEqual(creds.hashed, b"bar")
+
+
+Deprecation for whole packages or modules can be tested using the `importlib.reload` helper.
+
+.. code-block:: python
```

Comment:
For new code blocks, we should prefer `literalinclude` directives so that the Python code lives in Python files which can be loaded and tested.